src/frontend/src/forms/StockForms.tsx

```diff
@@ -283,6 +283,14 @@ function StockOperationsRow({
     input.removeFn(input.idx);
   };
 
+  const stockString: string = useMemo(() => {
+    if (!record.serial) {
+      return `${record.quantity}`;
+    } else {
+      return `#${record.serial}`;
+    }
+  }, record);
```

Comment:
@SchrodingersGat This should be:
```suggestion
  }, [record]);
```

Doesn't ts detect that issue?

Reply:
@wolflu05 nice catch as always - fix incoming in https://github.com/inventree/InvenTree/pull/7151